domains/authorization/Authorization.API/API/Authorization/[Features]/DeleteConsentCommand.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Data;
+using API.Repository;
+using API.ValueObjects;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Authorization._Features_;
+
+public class DeleteConsentCommandHandler(
+    IConsentRepository consentRepository,
+    IUserRepository userRepository,
+    IUnitOfWork unitOfWork)
+    : IRequestHandler<DeleteConsentCommand, bool>
+{
+    public async Task<bool> Handle(DeleteConsentCommand request, CancellationToken cancellationToken)
+    {
+        await unitOfWork.BeginTransactionAsync();
+
+        var consent = await consentRepository.Query()
+            .Include(c => c.Organization)
+            .FirstOrDefaultAsync(c => c.ClientId == request.ClientId && c.OrganizationId == request.OrganizationId, cancellationToken);
```

Comment:
Doesn't compute... request should contain IdpClientId. The ClientId is an internal database key.